docs/sources/panels-visualizations/panel-editor-overview/index.md

```diff
@@ -33,10 +33,32 @@ In the panel editor, you can update all the elements of a visualization includin
 
 ![Panel editor](/media/docs/grafana/panels-visualizations/screenshot-panel-editor-view.png)
 
-To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard. To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the dropdown:
+To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard. To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the drop-down:
 
 ![Add dropdown](/media/docs/grafana/dashboards/screenshot-add-dropdown-10.0.png)
 
+## Panel menu
+
+To access the panel editor, hover over the top-right corner of any panel. Click the panel menu icon that appears and select **Edit**. The panel menu gives you access to the following actions:
+
+- **View**: View the panel in full screen.
+- **Edit**: Open the panel editor to edit panel and visualization options.
+- **Share**: Share the panel as a link, embed, or library panel.
+- **Explore**: Open the panel in **Explore**, where you can focus on your query.
+- **Inspect**: Open the **Inspect** drawer, where you can review the panel data, stats, metadata, JSON, and query.
+  - **Data**: Open the **Inspect** drawer in the **Data** tab.
+  - **Query**: Open the **Inspect** drawer in the **Query** tab.
+  - **Panel JSON**: Open the **Inspect** drawer in the **JSON** tab.
+- **Extensions**: Access other actions provided by installed applications, such as declaring an incident. Note that this option doesn't appear unless you have app plugins installed which contribute an [extension](https://grafana.com/developers/plugin-tools/ui-extensions/) to the panel menu.
+- **More**: Access other panel actions.
+  - **Duplicate**: Make a copy of the panel. Duplicated panels query data separately from the original panel. You can use the special `Dashboard` data source to [share the same query results across panels][] instead.
+  - **Copy**: Copy the panel to the clipboard.
+  - **Create library panel**: Create a panel that can be imported into other dashboards.
+  - **Create alert**: [Create an alert][] directly from the panel.
```

Comment:
To me it reads like I can create an alert inline. I would make it clear for the user that the they will leave the dashboard view.